terracumber_config/tf_files/SUSEManager-4.2-beta-validation.tf

```diff
@@ -0,0 +1,973 @@
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-qam-42-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles11sp4", "sles12sp4o"]
+
+  mirror = "minima-mirror-qam.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "default"
+    bridge      = "br1"
+    additional_network = "192.168.41.0/24"
+  }
+}
+
+module "base_res" {
+  providers = {
+    libvirt = libvirt.caladan
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-qam-42-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "centos6o", "centos7o", "centos8o" ]
+
+  mirror = "minima-mirror-qam2.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "default"
+    bridge      = "br1"
+  }
+}
+
+module "base_newsle_ubuntu" {
+  providers = {
+    libvirt = libvirt.giediprime
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-qam-42-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles15o", "sles15sp1o", "sles15sp2o", "ubuntu1604o", "ubuntu1804o", "ubuntu2004o" ]
+
+  mirror = "minima-mirror-qam.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "default"
+    bridge      = "br1"
+    additional_network = "192.168.41.0/24"
+  }
+}
+
+module "server" {
+  source             = "./modules/server"
+  base_configuration = module.base_core.configuration
+  product_version    = "4.2-beta"
+  name               = "srv"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:89"
+    memory             = 40960
+    vcpu               = 10
+    data_pool            = "default"
+  }
+
+  repository_disk_size = 1500
+
+  auto_accept                    = false
+  monitored                      = true
+  disable_firewall               = false
+  allow_postgres_connections     = false
+  skip_changelog_import          = false
+  browser_side_less              = false
+  create_first_user              = false
+  mgr_sync_autologin             = false
+  create_sample_channel          = false
+  create_sample_activation_key   = false
+  create_sample_bootstrap_script = false
+  publish_private_ssl_key        = false
+  use_os_released_updates        = true
+  disable_download_tokens        = false
+  ssh_key_path                   = "./salt/controller/id_rsa.pub"
+  from_email                     = "root@suse.de"
+
+  //server_additional_repos
+
+}
+
+module "proxy" {
+  source             = "./modules/proxy"
+  base_configuration = module.base_core.configuration
+  product_version    = "4.2-beta"
+  name               = "pxy"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:8a"
```

Comment:
Good:
```
srv/salt/dhcpd-server/mgr_prv_suse_net/dhcpd.conf:host suma-qam-42-pxy
                { hardware ethernet aa:b2:92:42:00:8a; fixed-address 10.84.221.138; option host-name "suma-qam-42-pxy";               }
```
(although "qam" in the name is suboptimal... i just realized this now)

Reply:
It will be suboptimal for some months, then it will be good as we will test 4.2.x MU with exactly the same domainnames :P 
So is a kind of trade-offf, but I'm ok if you want to have another name for these months and then change it.